docs/Developers/Writing_Native_Bears.rst

```diff
@@ -598,4 +598,11 @@ For example, let's make an aspect bear named SpellingCheckBear.
                         aspect=DictionarySpelling('py'),
                     )
 
+.. note::
+
+    coala uses 1-based line & column convention, i.e., the first line
+    and first column are 1. However, some linters use 0-based convention.
+    For example, ``pylint`` uses 1-based line convention and 0-based column
```

Comment:
This is a bad example, as `pylint` is a linter, which has nothing to do with writing a **native** bear.
You might want to find an example among existing native bears

Reply:
I found, there are some bears in coala like - `PyLintBear` , `CheckstyleBear` ,`StylelintBear` .  they are normalized, but they all are linter bears.
 @li-boxuan is there any **native** bear that use 0 based convention. Btw i removed linter example.